app/src/main/res/values/strings.xml

```diff
@@ -175,4 +175,5 @@ Privacy Policy
     <string name="tracking_progress_message">Do not move the phone until tracking is completed</string>
     <string name="privacy_policy">Privacy Policy</string>
     <string name="select_tree_height_colour">Select tree height colour</string>
+    <string name="select_organization">Select organization</string>
```

Comment:
I don't think we need this if we just show a logo. Or perhaps we show it below the logo.

Reply:
Yeah under Greenstand logo which would be permanent there regardless of selected organisation.